terraform-modules/aws/airflow/default_iam_policy.json

```diff
@@ -38,13 +38,14 @@
                 "logs:GetQueryResults"
             ],
             "Resource": [
-                "arn:aws:logs:${aws_region}:${aws_account_id}:log-group:airflow-${airflow_name}-*"
+                "arn:aws:logs:${aws_region}:${aws_account_id}:log-group:airflow-${airflow_name}-*:*"
```

Comment:
"arn:aws:logs:${aws_region}:${aws_account_id}:log-group:airflow-${airflow_name}-*:*": This ARN represents all log groups with names starting with airflow-${airflow_name}- and all log streams within those log groups in the specified AWS region and account. The additional :* at the end of the ARN specifies that the policy covers both the log groups and their log streams.